HTML design changed

Kopeyka885 opened the pull request:
Deleted img tags, added buttons, sections and restructured the whole model

Kopeyka885 committed:
HTML design changed

Deleted img tags, added buttons, sections and restructured the whole model

fromanywhere commented on index.html:
```diff
@@ -1,46 +1,99 @@
+
+      <main class="todo-app__main-content main-content">
+
+        <section class="main-content__search-input search-input">
```
Оно здесь не search, оно здесь post, add

Kopeyka885 replied:
Исправил.

fromanywhere commented on index.html:
```diff
@@ -1,46 +1,99 @@
+            </li>
+
+            <li class="tasks-list-item list-item">
+              <div class="list-item__view">
```
Если здесь редактируемое поле лежит в плоском списке, то неясно, в чем функция list-item__view

Kopeyka885 replied:
Добавил дополнительные div'ы для input'a, как было показано в материалах лекции

Kopeyka885 committed:
Classes renaming, div's adding

Classes in first section are renamed, new divs added in list items (like in lection presentation).

Kopeyka885 reviewed (commented):
Добавил commit с комментариями об изменениях

fromanywhere commented on index.html:
```diff
@@ -1,46 +1,112 @@
+          <li class="tasks-list-item list-item">
+            <div class="list-item__view">
+              <input type="checkbox" class="list-item__input_checkbox" />
+              <span class="" tasks-item__text>JS</span>
```
атрибут вылетел из кавычек

fromanywhere commented on index.html:
```diff
@@ -1,46 +1,112 @@
 <head>
-	<meta charset="UTF-8">
-	<title>todos</title>
+  <meta charset="UTF-8" />
```
С форматированием конечно так себе

Kopeyka885 committed:
Code formating and class name fix

Вернул табуляцию в 4 знака и исправил вылезший за кавычки класс
